views/items/index.pug

```diff
@@ -2,3 +2,8 @@ extends ../layouts/authenticated-layout
 
 block content
   Items
+
+  form(method="post", action="/items/create")
```

Comment:
See note above about changing route - this will change the action here to "/items"